src/app/DIAmeterApplication.cpp

```diff
@@ -0,0 +1,558 @@
+#include <cstdio>
```

Comment:
For this entire file. Indentation is 2 space instead of a tab. This will follow what the rest of Crux does.